connector/spanmetricsconnector/config.go

```diff
@@ -71,6 +71,7 @@ type Config struct {
 }
 
 type HistogramConfig struct {
+	Unit        string                      `mapstructure:"unit"`
```

Comment:
I would use an enum type with `TextUnmarshaler` here, similar to https://github.com/open-telemetry/opentelemetry-collector/blob/2729a2ec661d590e948a4435ebffc32f624e17c7/config/configtelemetry/configtelemetry.go#L42. I am going to merge this since the component is still in development and we can therefore do this in a follow up PR without breaking changes, but would you mind creating an issue?